lib/util/include/hse/util/parse_num.h

```diff
@@ -94,21 +94,28 @@ parse_long(const char *str, long *result);
 
 /* definitions */
 __parse_unsigned_func(parse_u8, uint8_t, (uint8_t)0, UINT8_MAX)
-__parse_signed_func(parse_s8, int8_t, INT8_MIN, INT8_MAX)
+    __parse_signed_func(parse_s8, int8_t, INT8_MIN, INT8_MAX)
```

Comment:
Wowee this needs changing.

Reply:
Is this missing a bunch of semicolons?  How did it ever compile?
Also, need to stop using leading double underscores, those are always reserved for use by the implementation.  clang now has an option to warn about that...

Reply:
added off/on coments